src/screentovnc.cpp

```diff
@@ -972,64 +995,64 @@ void ScreenToVnc::mouseHandler(int buttonMask, int x, int y, rfbClientPtr cl)
             event_end.code = SYN_REPORT;
             event_end.value = 0;
 
-            if (hasAbsMtTrackingId){
+            if (hasAbsMtTrackingId) {
                 event_mt_tracking_id.type = EV_ABS;
                 event_mt_tracking_id.code = ABS_MT_TRACKING_ID;
                 event_mt_tracking_id.value = 0xffffffff;
             }
 
-            if (hasBntTouch){
+            if (hasBntTouch) {
                 event_btn_touch.type = EV_KEY;
                 event_btn_touch.code = BTN_TOUCH;
                 event_btn_touch.value = 0;
             }
 
-            if (hasAbsMtTouchMajor){
+            if (hasAbsMtTouchMajor) {
                 event_mt_touch_major.type = EV_ABS;
                 event_mt_touch_major.code = ABS_MT_TOUCH_MAJOR;
                 event_mt_touch_major.value = 0;
             }
 
-            if (hasAbsMtWidthMajor){
+            if (hasAbsMtWidthMajor) {
                 event_mt_width_major.type = EV_ABS;
                 event_mt_width_major.code = ABS_MT_WIDTH_MAJOR;
                 event_mt_width_major.value = 0;
             }
 
-            if (hasBntTouch){
-                if(write(eventDev, &event_btn_touch, sizeof(event_btn_touch)) < sizeof(event_btn_touch)) {
+            if (hasBntTouch) {
+                if (write(eventDev, &event_btn_touch, sizeof(event_btn_touch)) < sizeof(event_btn_touch)) {
```

Comment:
Are there no signed/unsigned warnings from these? afaik ssize_t from write() vs size_t from sizeof() -> signed value is promoted to unsigned for comparing -> -1 return values becomes something that is unlikely to be caught as an error...
Or am I missing something? Like is write() here something else than ::write()?

Reply:
There are but I just thought the fixes currently would be sufficient for this round :)